pkg/agentfs/docker.go

```diff
@@ -51,94 +54,249 @@ func CreateDockerfile(dir string, settingsMap map[string]string) error {
+
+Supported project types:
+  • Python: requires requirements.txt or pyproject.toml
+  • Node.js: requires package.json
+
+Please ensure your project has the appropriate dependency file, or create a Dockerfile manually in the current directory`)
+	}
+
+	// Provide user feedback about detected project type
+	switch projectType {
+	case ProjectTypeNode:
+		fmt.Printf("✔ Detected Node.js project (found %s)\n", util.Accented("package.json"))
+		fmt.Printf("  Using template [%s] with npm/yarn support\n", util.Accented("node"))
+	case ProjectTypePythonUV:
+		fmt.Printf("✔ Detected Python project with UV package manager\n")
+		fmt.Printf("  Using template [%s] for faster builds\n", util.Accented("python.uv"))
+		// Validate UV project setup
+		validateUVProject(dir)
+	case ProjectTypePythonPip:
+		fmt.Printf("✔ Detected Python project with pip package manager\n")
+		fmt.Printf("  Using template [%s]\n", util.Accented("python.pip"))
 	}
 
 	var dockerfileContent []byte
 	var dockerIgnoreContent []byte
-	var err error
 
-	dockerfileContent, err = fs.ReadFile("examples/" + projectType + ".Dockerfile")
+	dockerfileContent, err = fs.ReadFile("examples/" + string(projectType) + ".Dockerfile")
 	if err != nil {
-		return err
+		return fmt.Errorf("failed to load Dockerfile template '%s': %w", string(projectType), err)
 	}
-	dockerIgnoreContent, err = fs.ReadFile("examples/" + projectType + ".dockerignore")
+
+	// Load the appropriate dockerignore template for each project type
+	dockerIgnoreContent, err = fs.ReadFile("examples/" + string(projectType) + ".dockerignore")
 	if err != nil {
-		return err
+		return fmt.Errorf("failed to load .dockerignore template for '%s': %w", string(projectType), err)
 	}
 
-	if projectType == "python" {
-		dockerfileContent, err = validateEntrypoint(dir, dockerfileContent, projectType, settingsMap)
+	if projectType.IsPython() {
+		dockerfileContent, err = validateEntrypoint(dir, dockerfileContent, dockerIgnoreContent, projectType, settingsMap)
 		if err != nil {
-			return err
+			return fmt.Errorf("failed to validate Python entry point: %w", err)
 		}
 	}
 
 	err = os.WriteFile(filepath.Join(dir, "Dockerfile"), dockerfileContent, 0644)
 	if err != nil {
-		return err
+		return fmt.Errorf("failed to write Dockerfile: %w", err)
 	}
 
 	err = os.WriteFile(filepath.Join(dir, ".dockerignore"), dockerIgnoreContent, 0644)
 	if err != nil {
-		return err
+		return fmt.Errorf("failed to write .dockerignore: %w", err)
 	}
 
+	fmt.Printf("\n✔ Successfully generated Docker files:\n")
+	fmt.Printf("  %s - Container build instructions\n", util.Accented("Dockerfile"))
+	fmt.Printf("  %s - Files excluded from build context\n", util.Accented(".dockerignore"))
+	fmt.Printf("\nNext steps:\n")
+	fmt.Printf("  ► Review the %s and uncomment any needed system packages\n", util.Accented("Dockerfile"))
+	fmt.Printf("  ► Build your agent: docker build -t my-agent .\n")
+	fmt.Printf("  ► Test locally: docker run my-agent\n")
```

Comment:
This typically won't work, as when WE run the container, we do secret injection. If we want to keep this, at the very least it will need to be `docker run --env-file .env.local my-agent`, but there's no guarantee they have the required LK secrets in their env file.